src/js/containers/award/awards/AwardDescription.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+  * AwardContainer.jsx
```

Comment:
Could you rename this file `AwardDescriptionContainer.jsx` to match naming convention? I would also rename the `awards` folder to `contracts` to match the components directory structure.